Tribler/Test/Core/base_test.py

```diff
@@ -1,5 +1,12 @@
 from Tribler.Test.test_as_server import AbstractServer
 
 
+class MockObject(object):
```

Comment:
You (and I) may had to introduce this earlier, we now make custom objects in tests already.